guide/glossary.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: guide
+title: Glossary
+nav_order: 4
+---
+
+# Glossary
+{: .no_toc }
+
+## Address
+
+An identifier of 26-35 alphanumeric characters that can be used to receive bitcoin. There are [several address formats](https://en.bitcoin.it/wiki/List_of_address_prefixes){:target="_blank"} based on different standards and use cases. Applications should make it clear to users which standards are being used. Support for formats varies, so users may need to know this information for backup and future recovery.
+
+When users enter addresses, these formats can also be used to instantly validate correctness and possibly deduct other useful information.
+
+These are the three most common ones:
+
+
+**Legacy address - [P2PKH](https://en.bitcoin.it/wiki/Transaction#Pay-to-PubkeyHash){:target="_blank"}**<br/>
+Stands for pay-to-pubkey-hash, i.e pay to a hash of the recipient’s public key. This was the original address format for bitcoin and is sometimes called *legacy address*. Legacy addresses are not segwit compatible, but you can still send BTC from a P2PKH address to a segwit address without any problems. The average fee when sending from a P2PKH address is likely to be higher than when sending from a segwit address, however, because legacy address transactions are larger in size.
```

Comment:
Not sure what `segwit compatible` means at this point. I think I can figure it out from reading the other definitions, but not totally sure and I have to read between the lines.

Can we simplify these last two sentences to just say that sending from Legacy addresses is a bit more expensive than sending from SegWit addresses, because they can't be stored/compressed as efficiently? I'd be interested in how big those savings are. If it's 0.0001%, then maybe need to say that as it should not be a major consideration.

Should we make some type of recommendation that users should migrate wallets to SegWit because legacy addresses may not be supported in the future, or anything if that sort?

Reply:
Also, I'd consistently use the term `Legacy` address over `P2PKH`, which is much harder to parse.

Reply:
Simplified by cutting the part that was not related to legacy